sync/sync.go

```diff
@@ -325,10 +325,23 @@ func (t *Task) isRevisionCountAndChainSame(fromClient, toClient *replicaClient.R
 		fromClient.GetAddress(), rwReplica.RevisionCounter, toClient.GetAddress(),
 		curReplica.RevisionCounter)
 	logrus.Infof("RW replica chain: %v, cur replica chain: %v", rwReplica.Chain, curReplica.Chain)
```

Comment:
lets print checkpoints also here

Reply:
printed checkpoint in if condition, which will be printed if checkpoint is not nil